crypto/perlasm/x86_64-xlate.pl

```diff
@@ -115,7 +115,10 @@
 	.long 4f - 1f
 	.long 5
 0:
-	.asciz "GNU"
+	.byte 0x47
```

Comment:
a comment that this is "GNU" is appropriate to have here ... for future people who look at the code

Reply:
Done.